custom/icds_reports/migrations/0016_add_v2_indicators.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
```

Comment:
You'll probably need to rebase branch this onto master and rename this migration and change the dependency